src/controllers/__tests__/DrawingModeController-test.js

```diff
@@ -5,10 +5,6 @@ import DrawingModeController from '../DrawingModeController';
 import * as util from '../../util';
 import {
     STATES,
-    SELECTOR_ANNOTATION_BUTTON_DRAW_CANCEL,
```

Comment:
Are these constants still used anywhere? If not, let's go ahead and remove them as well.